nexus/db-model/src/ip_pool.rs

```diff
@@ -28,13 +28,7 @@ pub struct IpPool {
     #[diesel(embed)]
     pub identity: IpPoolIdentity,
 
-    /// An optional ID of the project for which this pool is reserved.
-    pub project_id: Option<Uuid>,
-
-    /// An optional ID of the rack for which this pool is reserved.
-    // TODO(https://github.com/oxidecomputer/omicron/issues/1276): This
-    // should probably point to an AZ or fleet, not a rack.
-    pub rack_id: Option<Uuid>,
+    pub internal_only: bool,
```

Comment:
Would be nice to have a comment explaining what this means.

Reply:
Done: 1ea2005001863d89da8d37316314f0109b196caa